docs/api-docs/apps/guide/apps-03-developing.md

```diff
@@ -86,8 +73,10 @@ Once you've exposed an app to the internet, you can register a draft app in the
 
 
 ## Installing a draft app
-Draft apps are installable on stores owned by the same email as the [Developer Portal](https://devtools.bigcommerce.com/my/apps) account. Use the following steps to installLog into the store and navigate to **Apps** > **My Apps** > [**My Draft Apps**](https://login.bigcommerce.com/deep-links/manage/marketplace/apps/my-apps/drafts) and click the app to install the draft. Once you click **Install**, BigCommerce will begin the OAauth flow by making a `GET` request to the app's `/auth` callback URL. If the app handles all the requests successfully, the app will be installed and you can begin feature development.
-
+Any store registered to the same email as your [Developer Portal](https://devtools.bigcommerce.com/my/apps) account can install your draft apps. Use the following steps to install.  

```

Comment:
```suggestion
Any store registered to the same email as your [Developer Portal](https://devtools.bigcommerce.com/my/apps) account can install your draft apps. Use the following steps to install:

```

Reply:
Sorry, I saw Mark's first!  Change made in [commit 5f7deb0](https://github.com/bigcommerce/dev-docs/pull/1337/commits/5f7deb0792a0ddb74f72c3bd57b42303140b8273).